lib/hooks/useImageUploader.js

```diff
@@ -77,7 +82,15 @@ export const useImageUploader = ({ isMulti, mockImageGenerator, onSuccess, onRej
             count: rejectedFiles.length,
             files: rejectedFiles.map(({ file }) => file.name).join(', '),
           });
-          const errorMsg = get(rejectedFiles[0], 'errors.0.message') || '';
+
+          const availableExtensions = Object.values(DROPZONE_ACCEPT_ALL).flat().join(', ').toUpperCase();
+          const [firstRejectedFile] = rejectedFiles;
+          const [firstError] = firstRejectedFile.errors;
+          const { code, message } = firstError;
+          const errorMsg = msg[code]
+            ? intl.formatMessage(msg[code], { ...I18nFormatters, availableExtensions })
+            : message;
+
```

Comment:
The error message looks good, but how can we be sure that the files were rejected because of their extensions here? It sounds like we're missing something like:

```es6
if (allFilesRejectedBecauseOfType(rejectedFiles)) {
  errorMsg = ...;
} else {
  errorMsg = get(rejectedFiles[0], 'errors.0.message') || ''; 
}
```

Reply:
@Betree We have `rejectedFiles`:
![console2](https://user-images.githubusercontent.com/48645737/216370741-2706e537-a388-40b4-834d-b6848aa0616e.png)
Need to change message with 'file-invalid-type' code. 
Added property 'file-invalid-type' inside `msg`. And I compare `code` from `rejectedFiles` with properties from `msg` (`msg[code]`). If it matches, then I show the modified message, otherwise a default message. My solution not good?

Reply:
It is; thanks for providing the error details!